src/main/kotlin/no/nav/api/oppfolging/Nom.kt

```diff
@@ -24,10 +24,10 @@ class Nom(
         )
         .build()
 
-    val client: NomClient = if (isNotProd()) {
+    fun client(token: String): NomClient = if (isNotProd()) {
```

Comment:
Denne kan ikke være avhengig av `token` siden da må en ny `NomClient` lages for vært token vi ser i appen.
Det er ikke supert med tanke på ressursbruk og connectionpool, men også caching fra `CachedNomClient` vil bli borte ved en sånn tilnærming.